examples/configure_beam_simulation.py

```diff
@@ -0,0 +1,126 @@
+"""
+===================================
+Configure a Beam Heating Simulation
+===================================
+
+This example demonstrates a typical configuration of a HYDRAD
+simulation with electron beam heating.
+"""
+import pathlib
+import tempfile
+
+import astropy.units as u
+
+from pydrad.configure import Configure
+from pydrad.configure.data import get_defaults
+from pydrad.configure.util import get_clean_hydrad
+
+#################################################################
+# HYDRAD can be used for simulations of solar flares, in addition
+# to active region loops.  Since flares have significantly stronger
+# energy release, however, some of the parameters need to be set
+# more stringently than might be done for e.g. nanoflare heating.
+# This example will show how to set up a reasonable flare simulation.
+#
+# Let's start with the defaults:
+config = get_defaults()
+
+# Configure the basic parameters of the simulation first.  Let's assume a
+# loop length of 50 Mm, and run the simulation for one hour of simulation time.
+config['general']['loop_length'] = 50 * u.Mm
+config['general']['total_time'] = 3600 * u.s
+
+# We typically assume that the loop is heated by electron beam heating,
+# where electrons are accelerated to tens of keV in energy in the corona,
+# and stream towards the chromosphere, where they deposit that energy.
+#
+# With beam heating, we typically want to use a slightly more detailed
+# chromosphere (rather than assuming constant temperature).
+# This is for two reasons: (1) it needs to be dense enough to stop the
+# beam, and (2) the energy deposition will produce a more realistic
+# chromospheric evaporation.
+# To do this, we turn on optically thick radiation, which will calculate
+# optically thick radiative losses in the chromosphere, produced by
+# hydrogen, magnesium, and calcium (based on the formulation by
+# Carlsson & Leenaarts 2012).
```

Comment:
Doesn't have to be done in this PR, but we should store citations in a bib file and cite things using `sphinxcontrib-bibtex` similar to what is done in fiasco. I forgot that that was not enabled here.

Reply:
Yes, I totally agree.  There are a few other citations we could throw in this example as well.  

I'll create an issue for this.  I'm not terribly familiar with sphinx, so it would take me a bit longer to get that implemented.